indico/htdocs/js/indico/modules/designer.js

```diff
@@ -34,6 +34,9 @@
     var initialWidth = null;
     var initialHeight = null;
     var showConfirmationDialog = false;
+    var confirmMessage = $T.gettext("The back side of the templates that use this one as the back side will be removed.");
+    var confirmTitle = $T.gettext("Disassociate back sides");
+    var removeBackSide = false;
```

Comment:
`removeBackside` (we use `backside` and not `back_side` / `backSide` in other places)